packages/share_plus/share_plus/ios/Classes/FLTSharePlusPlugin.m

```diff
@@ -368,6 +370,13 @@ + (void)shareFiles:(NSArray *)paths
                                                 mimeType:mimeType
                                                  subject:subject]];
   }
+  if (text != nil) {
+    NSObject *data = [[NSURL alloc] initWithString:text];
+    if (data == nil) {
```

Comment:
Sorry, I don't understand this code block. `data` is never going to be `nil`, right?

Reply:
no data can be nill if the text is not valid NSURL
in this case (text is not valid NSURL) we create a SharePlusData with the given text

Reply:
got it, thanks!